handlers/generic_file_upload_ce_attachment_v1/README.md

```diff
@@ -0,0 +1,46 @@
+# Kinetic Request CE Attachment Upload With API
+    This handler is used to copy an attachment from one submission to another.
+
```

Comment:
Should document the usage of the Info Value properties like we are doing with the Parameters.